tests/integration/cli/tests/run.rs

```diff
@@ -263,104 +274,117 @@ fn run_wasi_works_non_existent() -> anyhow::Result<()> {
     Ok(())
 }
 
-// FIXME: Re-enable. See https://github.com/wasmerio/wasmer/issues/3717
-#[ignore]
 #[test]
 fn run_test_caching_works_for_packages() {
     // we're testing the cache, so we don't want to reuse the current user's
     // $WASMER_DIR
     let wasmer_dir = TempDir::new().unwrap();
-    let rust_log = [
-        "wasmer_wasix::runtime::resolver::wapm_source=debug",
-        "wasmer_wasix::runtime::package_loader::builtin_loader=debug",
-    ]
-    .join(",");
 
     let assert = Command::new(get_wasmer_path())
         .arg("python/python")
-        .arg(format!("--mapdir=.:{}", asset_path().display()))
+        .arg(format!("--mapdir=/app:{}", asset_path().display()))
         .arg("--registry=wasmer.io")
-        .arg("test.py")
-        .env("WASMER_DIR", wasmer_dir.path())
-        .env("RUST_LOG", &rust_log)
+        .arg("/app/test.py")
+        .env("WASMER_CACHE_DIR", wasmer_dir.path())
+        .env("RUST_LOG", &*CACHE_RUST_LOG)
```

Comment:
This works? Why it doesn't when I try on php/php https://github.com/wasmerio/wasmer/issues/4027, is there something special to do on package itself?

Reply:
I'm not entirely sure why it doesn't work. There are loads of tests that use `--mapdir` so directory mapping definitely works. You can even mount to `/`, but it'll break packages that bundle their own files (e.g. python and its standard library). 

Maybe it's an issue with PHP's filesystem access?